src/modules/tensor/cpu/kernel/glitch.cpp

```diff
@@ -89,28 +89,31 @@ RppStatus glitch_u8_u8_host_tensor(Rpp8u *srcPtr,
         srcPtrChannel = srcPtrImage + (roi.xywhROI.xy.y * srcDescPtr->strides.hStride) + (roi.xywhROI.xy.x * layoutParams.bufferMultiplier);
         dstPtrChannel = dstPtrImage;
 
+        Rpp32u vectorIncrement, alignedLength;
+
         if ((srcDescPtr->c == 3) && (srcDescPtr->layout == RpptLayout::NHWC) && (dstDescPtr->layout == RpptLayout::NCHW))
         {
             Rpp8u *dstPtrRow;
             dstPtrRow = dstPtrChannel;
-            Rpp32u alignedLength = (((roi.xywhROI.roiWidth)/ 8) * 8) - 8;
+            vectorIncrement = 8;
```

Comment:
If this is a constant, assign only once in the beginning of func or use a #define so instruction cycles and registers are not wasted